src/components/Search/index.tsx

```diff
@@ -27,142 +27,145 @@ import { trackCustomEvent } from "../../utils/matomo"
 
 export const SearchIconButton = forwardRef<IconButtonProps, "button">(
   (props, ref) => (
-    <IconButton
-      ref={ref}
-      icon={<MdSearch />}
-      fontSize="2xl"
-      variant="icon"
-      _hover={{ svg: { fill: "primary.base" } }}
-      {...props}
-    />
+    <Button ref={ref} variant="secondaryGhost" px={1.5} {...props}>
+      <MdSearch />
+    </Button>
   )
 )
 
-const Search = forwardRef<{}, "button">((_, ref) => {
-  const searchButtonRef = React.useRef<HTMLButtonElement>(null)
-  const { isOpen, onClose, onOpen } = useDisclosure()
```

Comment:
Main change here was the removal of this line. It was moved to the parent component (the Nav).